modules/manage/pages/manage-security/manage-users-and-roles.adoc

```diff
@@ -131,28 +115,425 @@ on [.ui]*travel-sample*, the view is as follows:
+Note that by left-clicking within the row, you display options for making changes:
 
 [#security_view_with_edit_options]
 image::manage-security/securityViewWithEditOptions.png[,720,align=left]
 
 By left-clicking on [.ui]*Delete*, you delete the user.
-By left-clicking on [.ui]*Edit*, you bring up the [.ui]*Edit testUser* dialog,
+By left-clicking on [.ui]*Edit*, you bring up the [.ui]*Edit <username>* dialog,
 with the options to redefine username, full name, and roles (the content of this dialog is very similar to that of the [.ui]*Add New User* dialog, examined in detail above).
-The btn:[Reset Password] button only appears when the selected user is
+The *Reset Password* button only appears when the selected user is
 _locally_ defined.
 Left-clicking on the button brings up a dialog that allows redefinition of the
 user's password:
 
 [#reset_password]
 image::manage-security/resetPassword.png[,260,align=left]
 
-For a complete account of the roles you can allocate and their significance,
-see xref:learn:security/roles.adoc[Roles].
+[#adding-an-externally-authenticated-user]
+=== Adding an Externally Authenticated User
+
+An _externally authenticated_ user is not authenticated on Couchbase Server.
+Instead, they are authenticated on a server external to the cluster.
+This means that the user's password is defined and maintained externally.
+
+External authentication must be supported by xref:learn:security/authentication-overview.adoc#native-ldap-support[Native LDAP], xref:learn:security/authentication-overview.adoc#using-saslauthd[saslauthd], or xref:learn:security/authentication-overview.adoc#introduction-to-pam-based-authentication[PAM].
+Appropriate set-up procedures must have been completed prior to the addition of externally authenticated users.
+
+For the detailed steps required to establish external authentication by means of Native LDAP (which is the recommended mechanism), see xref:manage:manage-security/configure-ldap.adoc[Configure LDAP].
+
+To add an externally authenticated user, on the *Add New User* dialog, select the *External* option:
+
+image::manage-security/externalUserRadioButton.png[,100,align=middle]
+
+This removes the password-related fields from the dialog, since they are not required for the creation of an externally authenticated user.
+The fields for the user's names remain, and can be used.
+
+The view provided by the dialog's right-hand panel can be toggled, by means of the *Roles* and *Groups* tabs, which appear above the panel:
+
+image::manage-security/rolesAndGroupsToggle.png[,140,align=middle]
+
+The *Roles* option can be used for externally authenticated users, just as it is used for locally authenticated users.
+The only difference is that no password need be specified for externally authenticated users.
+
+The *Groups* option can only be used when _Native LDAP_ has been established as the means of external authentication, and _LDAP group-support_ has been enabled.
```

Comment:
This is not correct actually. Groups can be used for both local and external users. These are Couchbase groups (not ldap groups). Any couchbase user (doesn't matter local or external) can be explicitly added to Couchbase group.

If native LDAP is configured it becomes possible to use LDAP groups in order to assign roles to users without adding them to Couchbase groups explicitly. In this case users also become members of Couchbase groups but it happens implicitly (the mapping sits in LDAP server in this case). Please let me know if it doesn't make any sense to you.

Reply:
Thanks, I understand this now. I have changed this page accordingly, I hope in the right ways.